### PR TITLE
Add blog post - Automating SONiC switch provisioning and management with NetBox as a central source

### PR DESCRIPTION
**Pull request by matofeder**
Relates to https://github.com/SovereignCloudStack/issues/issues/689

### CODE REVIEW

File: _i18n/en/_posts/blog/2024-11-12-sonic.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+custom scripts that help automate configuration synchronization for SONiC devices, making it easier to keep network
+configurations consistent and compliant.
+
+NetBox has been enhanced to support configuration management for SONiC devices through two custom scripts that automate
+configuration tasks. These scripts `SONiC Config Diff` and `SONiC Config Sync` are designed to align SONiC device
+configurations with what’s defined in NetBox. Currently, these scripts are executed manually and offer limited
+functionality, but they set the groundwork for more comprehensive network management.
+
+```text
+                           +----------------------+
```
Comment (OgarOgarovic) [inline _i18n/en/_posts/blog/2024-11-12-sonic.md:152]: I suggest making this diagram more understandable like this:

```
                                 +----------------------+
                                 |         NetBox       |
                                 +----------------------+
                                /|   Sonic Config Diff  |\
                               / |   Sonic Config Sync  | \ 
                              /  +----------------------+  \
                             /              |               \
                            /               |                \
           +-------------------+    +-------------------+     +-------------------+
           |     SONiC SW 1    |    |     SONiC SW 2    | ... |     SONiC SW N    |
           +-------------------+    +-------------------+     +-------------------+
```

Comment (matofeder) [reply _i18n/en/_posts/blog/2024-11-12-sonic.md]: Thank you!, here is the fix 4f1966f01b36db1dfd0bfb78378cb9e2698260db

**Review by OgarOgarovic — APPROVED** [review]
LGTM

**Comment by scoopex** [comment]
I merged the current state to the "staging" branch, to have a preview at https://staging.scs.community/

**Comment by matofeder** [comment]
> I merged the current state to the "staging" branch, to have a preview at https://staging.scs.community/

I added some minor fixes https://github.com/SovereignCloudStack/website/pull/1045/commits/add902be648f9fb57db1b2786b3f64120656c2bc and merged it to the staging branch again.

**Comment by scoopex** [comment]
> Any reason not to merge this @scoopex , @OgarOgarovic ?

Not from my view.

**Comment by scoopex** [comment]
@maxwolfs : I wanted to wait for the preview at https://staging.scs.community/website/de/news/ but it does not appear there (the blogpost ist merged to staging). Any idea why that is the case?

**Comment by maxwolfs** [comment]
> @maxwolfs : I wanted to wait for the preview at https://staging.scs.community/website/de/news/ but it does not appear there (the blogpost ist merged to staging). Any idea why that is the case?

Maybe the staging branch has been deleted and another branch has been merged to staging? I have just done it again with this branch to staging. Should work once completed.

**Comment by maxwolfs** [comment]
The build process creates the html page out of it ([see here](https://github.com/SovereignCloudStack/website/blob/gh-pages-staging/tech/2024/11/12/sonic/index.html)), but it seems that the content of [gh-pages-staging](https://github.com/SovereignCloudStack/website/blob/gh-pages-staging/tech/2024/11/12/sonic/index.html) does not get properly deployed on the server? At least that would be my guess.

**Review by scoopex — APPROVED** [review]
LGTM

**Pull request merged**